library/src/main/java/pl/rspective/voucherify/android/client/api/VoucherifyApi.java

```diff
@@ -13,4 +14,7 @@
     @GET("/client/v1/validate")
     VoucherResponse validateVoucher(@QueryMap Map<String, String> queryParams);
 
+    @POST("/client/v1/redeem")
+    VoucherRedemptionResult redeemVoucher(@Query("code") String code, @Query("tracking_id") String trackingId);
```

Comment:
@LukaszWiktor I made this based on this documentation: https://voucherify.readme.io/docs/redeem-voucher-client-side

I don't see anything there regarding RedemptionContext and redeem gift vouchers. So. it is not updated or we should have only this simple redeem for client side?